src/test/java/io/debezium/connector/cassandra/CommitLogProcessorTest.java

```diff
@@ -29,26 +29,30 @@
 public class CommitLogProcessorTest extends EmbeddedCassandraConnectorTestBase {
     private CassandraConnectorContext context;
     private CommitLogProcessor commitLogProcessor;
+    private SchemaProcessor schemaProcessor;
 
     @Before
     public void setUp() throws Exception {
         context = generateTaskContext();
         commitLogProcessor = new CommitLogProcessor(context);
+        schemaProcessor = new SchemaProcessor(context);
+        schemaProcessor.initialize();
         commitLogProcessor.initialize();
     }
 
     @After
     public void tearDown() throws Exception {
         deleteTestOffsets(context);
         commitLogProcessor.destroy();
+        schemaProcessor.destroy();
         context.cleanUp();
     }
 
     @Test
     public void testProcessCommitLogs() throws Exception {
         int commitLogRowSize = 10;
         context.getCassandraClient().execute("CREATE TABLE IF NOT EXISTS " + keyspaceTable("cdc_table") + " (a int, b int, PRIMARY KEY(a)) WITH cdc = true;");
-        context.getSchemaHolder().refreshSchemas();
+        // context.getSchemaHolder().refreshSchemas();
```

Comment:
These commented out lines were just deleted on the `dd2` branch.  I presume we're okay to delete them, @smiklosovic ?